src/internal/testpachd/mock_transaction.go

```diff
@@ -4,11 +4,20 @@ import (
 	"fmt"
 
 	"github.com/pachyderm/pachyderm/v2/src/internal/transactionenv/txncontext"
-	"github.com/pachyderm/pachyderm/v2/src/pfs"
 	"github.com/pachyderm/pachyderm/v2/src/pps"
 )
 
 // This code can all go away if we ever get the ability to run a PPS server without external dependencies
```

Comment:
I usually add a `TODO` for stuff like this.

Reply:
When I wrote this, I was not optimistic that this would ever happen, and I still doubt it.

Reply:
It seems like it should either be a TODO or removed. I don't think it is really adding value otherwise.